apps/test/unit/gamelab/GameLabTest.js

```diff
@@ -66,6 +67,44 @@ describe('GameLab', () => {
     describe('After being injected with a studioApp instance', () => {
       beforeEach(() => instance.injectStudioApp(studioApp));
 
+      it('Rerun mutes sounds', () => {
```

Comment:
I think it's worthwhile to have these tests even though they include stubbing-out lots of functionality because it tests against regressions of the 'muteUrls' functionality - which could leading to the audio getting triggered in every preview.
I wonder if @joshlory or @islemaster knows of a cleaner way to test this small part of GameLab?

Reply:
:+1: No test too small! I don't mind that there's lots of stubbing here if it covers the code we're actually trying to check.  Longer-term, needing lots of stubs is a code smell for not decoupling things well and we might consider whether we can make this easier to test.